src/assets/config_files/usecases.json

```diff
@@ -545,6 +545,36 @@
              },
              "next": "ta_form"
           }
+       ],
+       "modelvalidation": [
+          {
+             "title": "Basal Ganglia - BluePyOpt Optimized Model Validation - Multiple HOF",
+             "description": "Validates single cell basal ganglia models optimized via BluePyOpt for various features offered by eFEL; models contain multiple hall of fame parameter sets",
+             "experience": ["all"],
+             "maturity": ["beta"],
+             "access": null,
+             "disabled": false,
+             "picture": {
+                "src": "https://raw.githubusercontent.com/appukuttan-shailesh/PyNN_testData/master/logo_vf_bluepyopt_efel.png",
+                "alt": "Logo of model validation: BluePyOpt to eFEL"
+             },
+             "next": "mv_form",
+             "dataprotected": true
+          },
+          {
+             "title": "Generic - BluePyOpt Optimized Model Validation",
+             "description": "Validates models optimized via BluePyOpt for various features offered by eFEL",
+             "experience": ["all"],
+             "maturity": ["beta"],
+             "access": null,
+             "disabled": true,
+             "picture": {
+                "src": "https://raw.githubusercontent.com/appukuttan-shailesh/PyNN_testData/master/logo_vf_bluepyopt_efel.png",
+                "alt": "Logo of model validation: BluePyOpt to eFEL"
+             },
+             "next": "mv_form",
```

Comment:
Can you try here to use `"next": "ta-form"` instead of `"next": "mv_form"` because I think you are not adding any functionality to the mv_form different from ta-form.

Reply:
Yes currently `mv_form` is identical to `ta_form`. We wanted to explore how we could customize this for the validation usecases. But for starters, we decided to just get a version working. In future, we are keen to see how/if we could extend this to accommodate other features.

If you think it is better to add `mv_form` in future when it is ready, I could reuse `ta-form` for now.

Reply:
Ok very good that you would like to experiment!
In this commit for Alex Kozlov I'm creating a model pages, so the user after clicking on the usecase goes to this models and from there I retrieve the notebook and I replace a placeholder in the notebook with the name of the model 
https://github.com/antonelepfl/usecases/commit/e2638c5f0df766fc02d0d50d71b2ea18ab95c589

Let me know if you have any question

Reply:
That sounds like just the thing for us! I will take a look and get back to you. Thanks!